sys/parsoid.js

```diff
@@ -440,25 +441,25 @@ PSP.getFormat = function(format, restbase, req) {
         }
     }
 
+    if (!self._okayToRerender(req)) {
```

Comment:
This drops the no-cache check, rendering those pages completely unaccessible. Could you reinstate that check?